pkg/utils/tags.go

```diff
@@ -82,3 +82,29 @@ func RemoveTags(obj interface{}, tags []string) error {
 	structTags.Set(res)
 	return nil
 }
+
+func HasTags[T *kong.Consumer](obj T, tags []string) bool {
```

Comment:
My changes didn't involve the kong import. Added that in a8d30fc0f8885f38ac857ea5f061c94cc1eb0b6b

Reply:
@GGabriele Can you please add the godoc to this `func`?

Reply:
done!